packages/common/src/events/api/orval/api/orval-events.ts

```diff
@@ -239,14 +239,16 @@ export interface ISearchEvents {
   entityIds?: string[];
   /** Array of associated entity types. Example: Hub Site Application,Hub Initiative,Hub Project */
   entityTypes?: EventAssociationEntityType[];
-  /** Array of event ids */
+  /** Array of event ids. Cannot be used with notIds. */
   eventIds?: string[];
   /** response data format. Example: csv,json */
   f?: EventsSearchFormat;
   /** esri geometry object */
   geometry?: ISearchEventsGeometry;
   /** Array of relation fields to include in response. Example: associations,creator,location,onlineMeeting,registrations */
   include?: GetEventsInclude[];
+  /** Array of event ids to exclude. Cannot be used with eventIds */
+  notIds?: string[];
```

Comment:
This all looks good to me. There is a little bit of inconsistency with naming conventions between `eventIds` and `notIds`. Given `eventIds` has been around for a while now, might be good to rename `notIds` to `notEventIds` for clarity.